### PR TITLE
Fixing date display

### PR DESCRIPTION
**Pull request by wiliajc87**

### CODE REVIEW

File: app/controllers/events_controller.rb [hunk 1]
```diff
@@ -63,6 +63,6 @@ def media_api
   private
 
   def event_params
-    params.require(:event).permit(:title, :frequency, :description, :album_link)
+    params.require(:event).permit(:title, :frequency, :description, :album_link, :date_of_event)
```
Comment (SStrausman) [inline app/controllers/events_controller.rb:66]: Please add corresponding seed data.

**Comment by tjmasterson** [comment]
looks good, we can look into having a drop down for the date but saving it as a string later

File: app/controllers/welcome_controller.rb [hunk 2]
```diff
@@ -1,3 +1,4 @@
+require 'date'
```
Comment (SStrausman) [inline app/controllers/welcome_controller.rb]: remove require

**Comment by SStrausman** [comment]
code reviewed

**Pull request merged**